src/hermez-account.js

```diff
@@ -81,4 +83,15 @@ module.exports = class HermezAccount {
         tx.fromAx = this.ax;
         tx.fromAy = this.ay;
     }
+
+    /**
+     * Sign withdraw-bjj
+     * @param {Object} inputs - Zk inputs  for the withdraw-bjj circuit
+     * @return {Scalar} Signature parameters.
+     */
+    signWithdrawBjj(inputs) {
+        const h = withdrawUtils.hashInputsWithdrawBjj(inputs);
+        const signature = eddsa.signPoseidon(this.rollupPrvKey, h);
+        return signature;
```

Comment:
I would add a blank space between among `return` statement and `signature` computation